modules/react/combobox/lib/ComboboxInput.tsx

```diff
@@ -69,7 +88,11 @@ export const useComboboxInput = composeHooks(
         }
       },
       onBlur(event: React.FocusEvent) {
-        model.events.hide(event);
+        if (model.state.nonInteractiveIds.length > 0) {
+          return;
+        } else {
+          model.events.hide(event);
+        }
```

Comment:
```suggestion
        if (model.state.nonInteractiveIds.length <= 0) {
          model.events.hide(event);
        }
        return;
```
Will this work instead of full if/else?

Reply:
i think so?